src/indexing.ts

```diff
@@ -57,6 +58,30 @@ export async function index(vectorStore: VectorStore, options: IndexingOptions)
   progress.stop();
 }
 
+export async function indexWikipedia(vectorStore: VectorStore, term: string) {
+  console.log(`Fetching wikipedia entry for term "${term}"`);
+  const directDoc = await fetchDocForTerm(term);
+  if (!directDoc) {
+    throw new Error(`No document found for ${term}`);
+  }
+  console.log(`Document found, ingesting into ${vectorStore.name} vector store`);
+  const documents = await chunk({ type: 'wikipediaExtract', content: directDoc });
+  const { data: embeddings } = await createEmbedding({ input: documents });
+
+  const vectorizedDocuments = zip(documents, embeddings).map(([document, embedding]) => ({
+    id: generateId(),
+    text: document,
+    embedding: embedding.embedding,
+    metadata: { term },
+  }));
+
+  const success = await vectorStore.add(vectorizedDocuments);
+
+  if (!success) {
+    throw new Error(`Failed up write ${term}'s wikipedia entry to vector store`);
```

Comment:
Failed to* write I think?

Nit: `term` will not always work with a `'s`, so I'd maybe rework the language to be more like:

```ts
new Error(`Failed to write wikipedia entry to vector store for term "${term}"`)
```